nservicebus/transactional-session/index.md

```diff
@@ -72,9 +99,11 @@ The transactional session feature requires a persistence in order to store outgo
 
 ## Transaction consistency
 
-To guarantee atomic consitency across database and message operations, the transactional session requires the [Outbox](/nservicebus/outbox) to be enabled.
+To guarantee atomic consistency across database and message operations, the transactional session requires the [Outbox](/nservicebus/outbox) to be enabled. This combination of features provides the strongest consistency guarantees and is, therefore, the recommended, safe-by-default configuration.
+
+NOTE: The outbox has to be [enabled explicitly](/nservicebus/outbox/#enabling-the-outbox) on the endpoint configuration.
 
-With the Outbox disabled, database and message operations are not executed until the session is committed. All database operations share the same database transaction but message operations are not guaranteed to be atomic with the database changes. This might lead to phantom records or ghost messages when in case of a failure during the commit phase.
+With the Outbox disabled, database and message operations are not executed until the session is committed. All database operations share the same database transaction and are committed first. When the database operations are successful, the message operations are [batch-dispatched by the transport](/nservicebus/messaging/batched-dispatch.md). The message operations are not guaranteed to be atomic with the database changes. This might lead to phantom records or ghost messages in case of a failure during the commit phase.
```

Comment:
```suggestion
With the Outbox disabled, database and message operations are not applied until the session is committed. All database operations share the same database transaction and are committed first. When the database operations complete successfully, the message operations are [batch-dispatched by the transport](/nservicebus/messaging/batched-dispatch.md). The message operations and the database changes are not guaranteed to be atomic. This might lead to phantom records or ghost messages in case of a failure during the commit phase.
```